NuGet/Realm.Database/Realm.Database.targets

```diff
@@ -41,6 +41,14 @@
       <LinkerFlags>-lz -framework Security</LinkerFlags>
     </NativeReference>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == 'Xamarin.Mac'">
+    <NativeReference Include="$(_RealmNugetNativePath)..\runtimes\osx.10.10-x64\native\librealm-wrappers.dylib">
+      <Kind>Dynamic</Kind>
```

Comment:
That comes into play when we statically link a library that contains C++code that needs the C++ runtime shared library. Our wrappers already link it directly because they're a dynamic library.